main/integration-tests/__test__/api-tests/studies/update-study-api.test.js

```diff
@@ -0,0 +1,72 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const _ = require('lodash');
+const UpdateStudyFixture = require('./__fixtures__/update-study-fixture');
+const { getTestAdminClient } = require('../../../utils/auth-tokens');
+const { listStudies } = require('../../../utils/studies');
+const { updateStudyParams, getStudyParams } = require('../../../helpers/api-param-generator');
+
+describe('Update Study Test', () => {
+  let testFixture;
+
+  beforeAll(async () => {
+    testFixture = new UpdateStudyFixture();
+    if (!UpdateStudyFixture.baseReady || _.isUndefined(UpdateStudyFixture.baseReady)) await testFixture.setupParent();
+    if (!UpdateStudyFixture.ready || _.isUndefined(UpdateStudyFixture.ready)) await testFixture.setupPreRequisites();
+  });
+
+  describe('Update Study API', () => {
+    it('should fail while trying to update Open Data studies', async () => {
+      // BUILD
+      const adminClient = await getTestAdminClient(testFixture.testConfig);
```

Comment:
Instead of creating an `adminClient` and then passing `adminClient` to each function, what are your thoughts on creating a userClient, and passing the testConfig to the userClient
```
// userClient
class UserClient {
    let client;
    constructor(testConfig) {
       // initialize client
    }
    
    async listStudies(param) {
        //TODO: implement method
    }

}
```

So basically the idea is you pass the config into a `UserClient`. The client uses those credentials to make future API calls. You no longer hav to pass the `client` to each method.

Reply:
So one to consider here.. Future tests might depend on a different number of utility methods, therefore their fixtures, which would be responsible for creating a UserClient shown above, would need to populate them accordingly for each test suite. Passing the axoisClient helps keep this flexible. 

Let me know if I'm missing something.

Reply:
As discussed, adding a method in the test fixture for `getAdminUser` (similar to `createNonAdminUser`) which will return the user object along with the `axiosClient`